### PR TITLE
extend ga to track users

### PR DESCRIPTION
**Pull request by verybigman**
Solve for ga #22

### CODE REVIEW

**Comment by remnev** [comment]
@ixax look at this please

File: common.blocks/google-analytics/google-analytics.ru.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# google-analytics
+
+Блок для трекинга событий в гугл аналитике.
```
Comment (remnev) [inline common.blocks/google-analytics/google-analytics.ru.md]: We have an english only doc file. Can you translate to english?

File: package.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
     "bower": "1.4.1",
     "enb": "^0.15.0",
     "enb-bem-techs": "^1.0.4",
-    "enb-bem-tmpl-specs": "^0.11.0",
+    "enb-bem-tmpl-specs": "^0.11.2",
```
Comment (remnev) [inline package.json]: Why is it necessary? Or just BTW?

**Comment by remnev** [comment]
:+1: Nice solution!

But it doesn't solve gh-22 enough. We need the same ability for yandex-metrica. It's only a part of job we need to do. Thank you for contributing!

File: common.blocks/google-analytics/google-analytics.ru.md (hunk 1 above)
Comment (verybigman) [inline common.blocks/google-analytics/google-analytics.ru.md]: Yes!

File: package.json (hunk 2 above)
Comment (verybigman) [inline package.json]: it's not obligatory

**Comment by verybigman** [comment]
Yes, but we can't send events to ya without settings in web ui.

**Comment by verybigman** [comment]
ready to merge!:)

**Comment by remnev** [comment]
Wait. Link from README.md to the block doc is needed.

**Comment by verybigman** [comment]
@remnev you can?)

**Comment by remnev** [comment]
Yes, wait a little bit.

**Comment by zxqfox** [comment]
Great thing! Thanks guys :+1:

**Comment by remnev** [comment]
Please, squash commits to one and merge it to master without merge commit.

**Comment by verybigman** [comment]
@remnev do it)

**Pull request closed without merging**